creating-joy/pom.xml

```diff
@@ -20,14 +20,6 @@
 	</properties>
 
 	<dependencies>
-		<dependency>
-			<groupId>org.springframework.boot</groupId>
-			<artifactId>spring-boot-starter-data-jpa</artifactId>
-		</dependency>
-		<dependency>
-			<groupId>org.springframework.boot</groupId>
-			<artifactId>spring-boot-starter-security</artifactId>
```

Comment:
Why are JPA and security removed?

Reply:
Since I don't have the database setup, I can't run my application with JPA dependency. I added the spring security dependency again.